docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -346,25 +301,28 @@ __EOF__
 ----
 +
 ----
-oc patch openstackcontrolplane openstack --type=merge --patch-file=~/manila.patch
+$ oc patch openstackcontrolplane openstack --type=merge --patch-file=~/manila.patch
```

Comment:
Should “manila.patch” be a user-replaced value in this command? E.g.,

`$ oc patch openstackcontrolplane openstack --type=merge --patch-file=<manila.patch>`

* Replace `<manila.patch>` with the name of your patch file.